.devcontainer/create-tarball/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.238.0/containers/dotnet
+{
+	"name": "Source-Build Create Tarball",
```

Comment:
What do you think about naming this in reference to the end result rather than the steps performed during creation? e.g. 'Source-Build w/Tarball`

The other devcontainer could be `Source-Build w/Built Tarball`

It's too bad this name isn't used in the UI dev container drop down.  Do you know where it is used?

Given the dev container drop down uses the file path, it feels like the folder name should also reflect the end state.  I would also suggest moving the default dev container into a folder so that the folder name can give it context.  In a way this is actually a poor choice for non-source-build developers to use.

Reply:
I like your suggestion. The only place I see the name used is when using the devcontainer (not codespaces) locally in VS Code: 
<img width="214" alt="image" src="https://user-images.githubusercontent.com/36081148/177892978-1decc174-e11d-4ff9-8659-82b2608d6075.png">